pgex/examples/simple_projectile/main.py

```diff
@@ -0,0 +1,49 @@
+"""
+This file is a part of the 'Pygame Examples (pgex)' source code.
+The source code is distributed under the MIT license.
+"""
+
+import asyncio
+
+import pygame
+
+from .bullet import Bullet
+
+WIDTH, HEIGHT = 640, 360
+FPS = 60
+
+
+async def main() -> None:
+    pygame.init()
+    screen = pygame.display.set_mode((WIDTH, HEIGHT))
+    clock = pygame.time.Clock()
+
+    bullet_group = pygame.sprite.Group()
+
+    running = True
+    while running:
+        clock.tick(FPS)
+        screen.fill("black")
+
+        events = pygame.event.get()
+        for event in events:
+            if event.type == pygame.QUIT:
+                running = False
+            elif event.type == pygame.KEYDOWN:
+                if event.key == pygame.K_SPACE:
+                    new_bullet = Bullet(pygame.Vector2(0, 180), pygame.Vector2(5, 0))
```

Comment:
Oddly specific values could be store in a "const" identifier (or referred from an existing one).
It would bring two main advantages:
- Easier for readers to understand what are the point of these magic numbers (I don't know what `180` means, from what I can see it is the vertical starting position of the bullet but why this specific value? Is it the middle of the screen? Is it an arbitrary position for the bullet to be easily visible by the player?)
- Easier for coders to refactor/change the values